app/components/primer/beta/auto_complete.rb

```diff
@@ -95,8 +104,7 @@ class AutoComplete < Primer::Component
       #   <% end %>
       #
       # @example With Icon
-      #   <%= render(Primer::Beta::AutoComplete.new(src: "/auto_complete", list_id: "fruits-popup-4", input_id: "fruits-input-4", position: :relative)) do |c| %>
-      #     <% c.label(classes:"").with_content("Fruits") %>
+      #   <%= render(Primer::Beta::AutoComplete.new("aria-label": "Fruits", src: "/auto_complete", list_id: "fruits-popup-4", input_id: "fruits-input-4", position: :relative)) do |c| %>
```

Comment:
note API change
adding `aria-label="Your a11y description"`

Reply:
correction -- api is the same as before, and as we discussed with James this is likely not its final state but fine for now.